pagopa/io-app
# chore: [IA-915] Add `last_app_version` on every profile update request

## Short description
This PR is going to add the `last_app_version` field in every profile update request. A detailed description of _why_ this is needed is on the task.

## List of changes proposed in this pull request
- Add `last_app_version` on every profile update request.

## How to test
Try changing something in the profile, and the HTTP request should contain `last_app_version` with the right value. An example could be switching from the quick setup to the manual one in the service preferences.
